code/modules/mob/living/silicon/ai/death.dm

```diff
@@ -20,6 +20,7 @@
 	cameraFollow = null
 
 	set_anchored(FALSE) //unbolt floorbolts
+	status_flags |= CANPUSH //we want it to be pushable when unanchored on death
```

Comment:
```suggestion
	status_flags |= CANPUSH //we want it to be pushable when unanchored on death
	REMOVE_TRAIT(src, TRAIT_NO_TELEPORT, AI_ANCHOR_TRAIT)
```
this'd fix another AI bug while you're here

Reply:
does this bug have an open issue? if so i can link it to close